zerver/lib/push_notifications.py

```diff
@@ -603,6 +603,15 @@ def get_apns_alert_subtitle(message: Message) -> str:
     # For group PMs, or regular messages to a stream, just use a colon to indicate this is the sender.
     return message.sender.full_name + ":"
 
+def get_apns_badge_count(user_profile: UserProfile, read_messages_ids: Optional[List[int]]=[]) -> int:
+    return UserMessage.objects.filter(
```

Comment:
This deserves an explanatory comment on why we're excluding `read_message_ids` here.